models/review.js

```diff
@@ -1,36 +0,0 @@
-const { Schema, model } = require("mongoose");
-
-// Review Schema for the reviews being left by users about companies. It references the both the review and company schema.
-
-const contentSchema = new Schema({
-  reviewTitle: String,
-  overallScore: Number,
-  features: Number,
-  easyToUse: Number,
-  customerSupport: Number,
-  valueForMoney: Number,
-  distribution: Number,
-  proBullets: [String],
-  conBullets: [String],
-});
-
-const reviewSchema = new Schema(
-  {
-    content: contentSchema,
```

Comment:
My mongodb knowledge is a bit shakey but are you sure you need a `Schema` within a `Schema`? It seems a little fishy...